### ISSUE TITLE
SCRIPT438: Object doesn't support property or method 'getFiles' 

### ISSUE THREAD

**Issue by ProX666** [danialfarid/ng-file-upload#302]
Hi. I am trying to get it working on IE9. But i get the error: SCRIPT438: Object doesn't support property or method 'getFiles' 

I read a lot online, but can not get it working :(

This is what i have so far:

Loading order:
/js/angular-file-upload/angular-file-upload-shim.js
//ajax.googleapis.com/ajax/libs/angularjs/1.2.8/angular.js
/js/angular-file-upload/angular-file-upload.js

Form (i am only interested in "select a file from your computer" for now:
    <form name="profilePhoto" class="profilephoto-form" novalidate ng-submit="submitProfilePhoto()">
To attach an image drag & drop it here or 
<label>
<input type="file" class="ng-hide" ng-model="file" ng-file-select="onFileSelect($files)" />
{{'select a file from your computer' | trans}}
</label>
</form> 

Angular handling:
$scope.upload = $upload.upload({
                        url: '/json/dashboard/file-upload',
                        method: 'POST',
                        file: $scope.selectedFile,
                        fileFormDataName: 'file'
                    }).success(function(response) {
                        $scope.user.profilephoto = response.photo;
                        $scope.file = '';
                    }).error(function(response) {
                        console.log("error");
                    }).progress(function(evt) {
                        $scope.progress = parseInt(Math.min(100.0, 100.0 \* evt.loaded / evt.total));
                        console.log($scope.progress);
                    }); 

My error: SCRIPT438: Object doesn't support property or method 'getFiles' 
in angular-file-upload-shim.js, line 179 character 4

var changeFnWrapper = function(fn) {
        return function(evt) {
            var files = FileAPI.getFiles(evt);

I am not sure how to solve it. Do i miss something (or a lot)?

**Comment by mygzi** [comment]
Explicitly loading FileAPI.min.js solved this IE9 issue for me as well. All relevant files are at the same path, but that didn't seem to be enough. Working script load snippet:

```
<!--[if lt IE 10]>
<script src="/components-path/ng-file-upload/FileAPI.min.js"></script> 
<script src="/components-path/ng-file-upload/angular-file-upload-shim.min.js"></script> 
<![endif]-->
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.22/angular.min.js"></script>
<script src="/components-path/ng-file-upload/angular-file-upload.min.js"></script>
```

**Comment by bradw2k** [comment]
I had this problem, and found that the browser was trying to load FileAPI.js, not FileAPI.min.js which I had in the folder.  Simply renaming the file to FileAPI.js fixed the problem for me, did not have to resort to an explicit script tag for FileAPI.

**Comment by danialfarid** [comment]
@bradw2k fixed at 11.0.2
